### PR TITLE
Add travis and tox configuration.

### LINKED ISSUE #11: Configure travis
- [ ] Add tests with PyTest
- [ ] Configure tox
- [ ] Configure travis
- [ ] Add README badge for coveralls and travis.

### PR DESCRIPTION
**Pull request by Natim**
Fixes #11

### CODE REVIEW

File: tox.ini [hunk 1]
```diff
@@ -0,0 +1,20 @@
+[tox]
+envlist = py35,py36,flake8
+skip_missing_interpreters = True
```
Comment (magopian) [inline tox.ini]: that may be "dangerous": if for some reason an interpreter is missing on travis, we won't be warned, I think?

File: tox.ini [hunk 2]
```diff
@@ -0,0 +1,20 @@
...
+
+[testenv]
+commands =
+    python --version
+    py.test --cov-report term-missing --cov-fail-under 100 --cov pollbot {posargs}
+deps =
+    -rdev-requirements.txt
+
```
Comment (magopian) [inline tox.ini]: extraneous blank line

File: tox.ini [hunk 3]
```diff
@@ -0,0 +1,20 @@
...
+install_command = pip install --pre {opts} {packages}
+
+[testenv:flake8]
+commands = flake8 pollbot tests
+deps =
+    flake8
+
+[flake8]
+max-line-length = 99
```
Comment (magopian) [inline tox.ini]: yewwww

File: tox.ini (hunk 1 above)
Comment (Natim) [inline tox.ini]: This is more for people running tox on their machines than using travis. The travis config makes explicit which interpreter to use.

**Pull request merged**